### ISSUE TITLE
 RST-IVV-179: ICDH Build 2: cFS Tool Contains Potentially Null Pointers that are De-referenced in Function

### ISSUE THREAD

**Issue by n-yanchik** [nasa/elf2cfetbl#103]

**Comment by skliper** [comment]
@n-yanchik do you need this fixed for Draco?

**Comment by n-yanchik** [comment]
Nope. WFI FSW doesn’t need it. I don’t think we are using that tool. But we figured you should know about the issue.



Thanks,
Nicholas
301-286-7134 (forwarded while working from home)



From: Jake Hageman ***@***.***>
Reply-To: nasa/elf2cfetbl ***@***.***>
Date: Wednesday, February 23, 2022 at 2:08 PM
To: nasa/elf2cfetbl ***@***.***>
Cc: "Yanchik, Nicholas J. (GSFC-5820)" ***@***.***>, Mention ***@***.***>
Subject: [EXTERNAL] Re: [nasa/elf2cfetbl] RST-IVV-179: ICDH Build 2: cFS Tool Contains Potentially Null Pointers that are De-referenced in Function (Issue #103)


@n-yanchik<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fn-yanchik&data=04%7C01%7Cnicholas.j.yanchik%40nasa.gov%7C3044eae9267440163a8708d9f6ffe160%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637812401130921767%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=uSaFjjC%2B5V%2F6XgrR0WHn9yr3BnAsxzqjbxSEODsU5bE%3D&reserved=0> do you need this fixed for Draco?

—
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnasa%2Felf2cfetbl%2Fissues%2F103%23issuecomment-1049118862&data=04%7C01%7Cnicholas.j.yanchik%40nasa.gov%7C3044eae9267440163a8708d9f6ffe160%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637812401130921767%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=8vEJgPiWNjgWFTmlSzo8sp3bkSW7VNNjp8pBuqK8Saw%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAX5KA5OH4RYTERNEXYQQIR3U4UWCZANCNFSM5PFIA4KA&data=04%7C01%7Cnicholas.j.yanchik%40nasa.gov%7C3044eae9267440163a8708d9f6ffe160%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637812401130921767%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Lc7d8ZP8yMOQT8pckfRANRkYrlhQHOSSAQ04fKH7Vc0%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cnicholas.j.yanchik%40nasa.gov%7C3044eae9267440163a8708d9f6ffe160%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637812401130921767%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=poDLoelAGICpP5hge3sdr2%2F0Bs5ZWxhcLDJqumjXb%2Bw%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cnicholas.j.yanchik%40nasa.gov%7C3044eae9267440163a8708d9f6ffe160%7C7005d45845be48ae8140d43da96dd17b%7C0%7C0%7C637812401130921767%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=s%2BINLRAaVx0NAGY567tvYy8y%2Fr2HydO9KQ36cIPy7PE%3D&reserved=0>.
